test/onnx/test_pytorch_onnx_onnxruntime.py

```diff
@@ -7949,6 +7949,17 @@ def forward(self, x, window_length: int):
         output = module(x, win_length)
         self.run_test(module, (x, win_length))
 
+    @skipIfUnsupportedMinOpsetVersion(9)
+    @disableScriptTest()
+    def test_fill(self):
+        class FillModule(torch.nn.Module):
```

Comment:
What if the dtype for x and filled_val does not match? Do we need to cover this case?

Reply:
According to the behavior of torch.Tensor.fill_() method, it will keep the dtype of the original tensor so I've done similar work in the symbolic function by passing the dtype of input tensor into full_like() function to make sure the dtype of final outout is same as the input.